react/features/prejoin/components/web/Prejoin.js

```diff
@@ -400,6 +395,7 @@ class Prejoin extends Component<Props, State> {
                             content = { hasExtraJoinButtons && <div className = 'prejoin-preview-dropdown-btns'>
                                 {extraButtonsToRender.map(({ key, ...rest }: Object) => (
                                     <DropdownButton
```

Comment:
I suggest we replace it with the new UI Button web component and remove DropdownButton component because this is the only place that it's used.